src/utils/constants.js

```diff
@@ -20,6 +30,18 @@ const SUBPAGES = {
   GARRISON: 'Garrison',
 }
 
+const SUBPAGES_INDEX = {
+  [SUBPAGES.CITY]: 0,
+  [SUBPAGES.COLONY]: 1,
+  [SUBPAGES.RESEARCH]: 0,
+  [SUBPAGES.SPELLS]: 0,
+  [SUBPAGES.PRAYERS]: 1,
+  [SUBPAGES.ARMY]: 0,
+  [SUBPAGES.EXPLORE]: 1,
+  [SUBPAGES.ATTACK]: 3,
```

Comment:
Would that work if one doesn't have Espionage researched?

Reply:
![image](https://user-images.githubusercontent.com/108338093/223281533-69524d42-4fcd-45db-8457-f0f1de958014.png)